client/allocdir/fs_linux.go

```diff
@@ -88,3 +88,11 @@ func removeSecretDir(dir string) error {
 	}
 	return os.RemoveAll(dir)
 }
+
+func getOwner(fi os.FileInfo) (int, int) {
+	stat, ok := fi.Sys().(*syscall.Stat_t)
```

Comment:
Is `syscall.Stat_t` not implemented on the other platforms we chroot in? We shouldn't leave the bug open on other platforms we chroot in